### PR TITLE
Bug 1822515: 4.4 - tests: split API availability monitor into kube and OpenShift

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: test/extended/util/disruption/controlplane/controlplane.go [hunk 1]
```diff
@@ -4,33 +4,54 @@ import (
...
 
-// AvailableTest tests that the control plane remains is available
-// before and after a cluster upgrade.
-type AvailableTest struct {
+// KubeAvailableTest tests that the Kubernetes control plane remains available during and after a cluster upgrade.
+type KubeAvailableTest struct {
+	availableTest
+}
+
+func (KubeAvailableTest) Name() string        { return "kube-api-available" }
```
Comment (smarterclayton) [inline test/extended/util/disruption/controlplane/controlplane.go:21]: `kubernetes-api-available` just to be clear

Comment (smarterclayton) [reply test/extended/util/disruption/controlplane/controlplane.go:21]: Oh, this is your backport.  It's fine.

Comment (sttts) [reply test/extended/util/disruption/controlplane/controlplane.go]: updated. Now you have to retag this and https://github.com/openshift/origin/pull/24844

**Pull request merged**